src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1219,13 +1224,35 @@ def install_helm_client() -> str:
         logger.warning(
             "Downloading helm client for first time. This can take few minutes..."
         )
+        active_directory_array = cmd.cli_ctx.cloud.endpoints.active_directory.split(".")
```

Comment:
considering that this code is used in multiple places, can you move it into one function and reference that?